oak-store-spi/src/main/java/org/apache/jackrabbit/oak/json/JsonSerializer.java

```diff
@@ -79,17 +79,30 @@ public class JsonSerializer {
     private final BlobSerializer blobs;
 
     private final boolean catchExceptions;
+    private final PrintWriter printWriter;
+    public static final String WRITER_FLUSH_THRESHOLD_KEY = "oak.writerFlushThreshold";
+
+    private final long defaultWriterFlushThreshold = 100*1024*1024L; // 100 MB
```

Comment:
minor: improve code formatting
```suggestion
    private final long defaultWriterFlushThreshold = 100 * 1024 * 1024L; // 100 MB
```